src/Microsoft.Health.Fhir.Core/Features/Conformance/ListedConformanceStatementExtensions.cs

```diff
@@ -4,23 +4,19 @@
 // -------------------------------------------------------------------------------------------------
 
 using System.Collections.Generic;
+using System.Linq;
+using EnsureThat;
+using Microsoft.Health.Fhir.Core.Features.Conformance.Models;
 
 namespace Microsoft.Health.Fhir.Core.Features.Conformance
 {
-    public interface IListedResourceComponent
+    public static class ListedConformanceStatementExtensions
     {
-        bool? UpdateCreate { get; set; }
+        public static ListedRestComponent Server(this IEnumerable<ListedRestComponent> restComponents)
+        {
+            EnsureArg.IsNotNull(restComponents, nameof(restComponents));
 
-        bool? ConditionalUpdate { get; set; }
-
-        bool? ConditionalCreate { get; set; }
-
-        bool? ReadHistory { get; set; }
-
-        IList<string> SearchRevInclude { get; }
-
-        IList<string> SearchInclude { get; }
-
-        void AddResourceVersionPolicy(string policy);
+            return restComponents.Single(x => x.Mode == ListedCapabilityStatement.ServerMode);
```

Comment:
The `ListedCapabilityStatement.Rest` is already a hashset using `PropertyEqualityComparer` with Mode. Would that be utilized here when finding element. If not, can we utilize it?

Reply:
I don't think so? PropertyEqualityComparer is ensuring only 1 RestComponent of each Mode is added (Server/Client). This is explicitly fetching the Server rest component.